manifests/f/Foundertype/FounderClient/1.1.3/Foundertype.FounderClient.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Foundertype.FounderClient
+PackageVersion: 1.1.3
+PackageLocale: en-US
+Publisher: Founder
+PublisherUrl: https://www.foundertype.com/index.php/Index/index.html
+Author: Foundertype
+PackageName: Founder Client
+PackageUrl: https://www.foundertype.com/index.php/Index/index.html
+License: Proprietary
+LicenseUrl: https://www.foundertype.com/index.php/About/powerAllowPro.html
+Copyright: Copyright 2000-2023年 FounderType.Allrights reserved.
+CopyrightUrl: https://www.foundertype.com/index.php/About/powerAllowPro.html
+ShortDescription: Founder Client provided massive different kinds of high-quality fonts, and own excellent font sources from all over the world. Founder Client is aimed at providing more efficient and convenient font-using service
+Moniker: 方正字库

```

Comment:
This should be an english localized moniker. If a non en-US moniker is desired, the appropriate locale file should be added and set as the defaultLocale, and the en-US file should be set as a normal locale file